<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1434</width>
    <height>958</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>piratengine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1411</width>
      <height>901</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1381</width>
        <height>851</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QTableWidget" name="FilesTable"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="BackupDBButton">
           <property name="text">
            <string>Backup database</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CreatePlaylistButton">
           <property name="text">
            <string>Create playlist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddTrackToPlaylistButton">
           <property name="text">
            <string>Add track(s) to playlist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ImportToPlaylistButton">
           <property name="text">
            <string>Import file to playlist (txt,json,m3u)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExportPlaylistButton">
           <property name="text">
            <string>Export playlist(s) (txt,json,m3u)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ScanFolderButton">
           <property name="text">
            <string>Scan media for music files not in database</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ImportFilesButton">
           <property name="text">
            <string>Import file(s) to Track database</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QTableWidget" name="TrackTable">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTableWidget" name="PlaylistContentTable">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QTableWidget" name="PlaylistTable"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="FilesTableLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>185</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Files</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="FilesFilterLabel">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FilesFilter"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="TrackTableLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777142</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Tracks</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="TrackFilterLabel">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TrackFilter"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QTextEdit" name="DBPathTextEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DBLoadButton">
               <property name="text">
                <string>Load db</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DBChooseDirButton">
               <property name="text">
                <string>Choose dir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTableWidget" name="DBTable"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="PlaylistTableLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>86</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Playlists</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="PlaylistFilterLabel">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PlaylistFilter"/>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="PlaylistContentTableLabel">
             <property name="text">
              <string>Playlist Content</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="PlaylistContentFilterLable">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PlaylistContentFilter"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>StagelinQ</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>991</width>
        <height>631</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="stagelinqStartButton">
             <property name="text">
              <string>Start StageLinQ monitor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="stagelinqIPFilterLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stagelinqIP">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>460</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="stagelinqDataFilterLabel">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stagelinqDataFilter">
             <property name="toolTip">
              <string extracomment="Filter on stagelinq data"/>
             </property>
             <property name="text">
              <string extracomment="filter on stagelinq data"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="stagelinqTable"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>log</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1381</width>
        <height>841</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QTextBrowser" name="logTextBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1434</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_database"/>
    <addaction name="actionChoose_table_columns"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_database">
   <property name="text">
    <string>Load database</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChoose_table_columns">
   <property name="text">
    <string>Choose table columns</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
